MANIFEST.in

```diff
@@ -1,15 +1,2 @@
-# data
-include niworkflows/data/*.json
```

Comment:
After some struggle (I had `setuptools-scm` installed and that basically dismisses the ``MANIFEST.in``), and after having a look on fitlins' implementation I came up with this minimalistic version of the manifest.

I haven't figured out how to drop the `conftest.py` files on binary distributions, but I'm giving that up since it is fairly out of scope of this PR.